tests/pwm/pwmtestersut.ino

```diff
@@ -0,0 +1,42 @@
+#include <ESP8266WiFi.h>
+
+const int pin[] = { D0, D1, D2, D3, D4, D5, D6, D7, D8 };
+
+void setup() {
+  WiFi.disconnect(true);
+  WiFi.forceSleepBegin();
+  Serial.begin(115200);
+  Serial.printf("\nPWM tester\n");
+  Serial.printf("z        = Reset ESP8266\n");
```

Comment:
Instead of putting this here, I suggest an h command that prints it out.

Reply:
Done